### DISCUSSION TITLE
Welcome to Midios Discussions!

### DISCUSSION THREAD

**Discussion by tshoppa** [tshoppa/Midios#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sonotor** [comment]
Hello,
Great to discover your Midimix configurator. You did aa amazing job!
I've tried it and I like it, it works but I can't understand why there's something wrong with what I've written on the configurator.
I would like to have all the buttons transmitting on midi channel 09.
It's possible to  have all the faders on midi channel 09 but impossible to have the buttons  on midi channel 09! 
I can't change it, I've tried several times.... ???
Could you have a look on the files in attachment and tell me why I cannot have what I want?
I spent few hours trying  and testing...
thank you in advance
Regards!
![Capture d’écran 2025-01-26 à 17 40 20](https://github.com/user-attachments/assets/b4fdfaef-514d-45d1-a93f-87f0fdd68fcc)
![Capture d’écran 2025-01-26 à 17 40 15](https://github.com/user-attachments/assets/178549c5-8652-450b-83e4-3e9224faa4dd)
[midimix gab finalv1.sbr.zip](https://github.com/user-attachments/files/18550898/midimix.gab.finalv1.sbr.zip)

**Comment by tshoppa** [comment]
Hi,
you are right, setting the channel for buttons doesn't work, it's always
ch1. Moreover, I couldn't even set it to send cc, it's always nt (note).
I will have a look at it, but since I haven't touched it for years I cannot
promise that I can fix it... (maybe if it is in the configurator, but the
bytestreamer code is like machine code and really hard to understand once
you're off :-P)
best regards,
Stefan

Am So., 26. Jan. 2025 um 18:02 Uhr schrieb sonotor ***@***.***

**Comment by sonotor** [comment]
Thank you very much!
Very kind of you.
Merci beaucoup!


Le 1 févr. 2025 à 17:02, à 17:02, Schulzki ***@***.***> a écrit:

**Comment by tshoppa** [comment]
Ah, tu es francais? Haha, je suis allemand et je fais toutes mes vacances
dans "les vans"
Sorry, my french is bad, but I am in France with my family every year.
I think I've found the bug. I will fix it tomorrow.
cordialement

Am Sa., 1. Feb. 2025 um 19:41 Uhr schrieb sonotor ***@***.***

**Comment by tshoppa** [comment]
Ok, it should work now, please try again

best,
Stefan

Am Sa., 1. Feb. 2025 um 21:19 Uhr schrieb Stefan Schulz <
***@***.***>:

**Comment by sonotor** [comment]
Dear Stefan,
GREAT!!!!!!!!!
C'est fantastique!
Just a quick check and It seems to work perfectly well.
I will try more deeply in the next few days.
I'm so happy!!!!
C'est génial.
And.... je ne suis pas en France mais en Suisse pas loin de la frontière française.
Toi tu vas à Les Vans en Ardèche? C'est beau!
mais pour discuter c'est peut-être mieux de le faire sur un email privé....
Encore merci!!!!
Gab

**Comment by sonotor** [comment]
Hi Stefan,
I hope to find you well.
I tried different configurations and it's GREAT!!!
Everything works very well except one button:
bank right / the rec arm button nbr 8 
It doesn’t send correctly the channel midi selected and is by default lightened .

So there is probably still a small error in the code.

I share to you the .sbr file showing the problem.

When you go to the right bank you will see that the nr.8 rec arm button will be lightened  and it won't send the right data.

Thanks in advance and.... once again you did an amazing good job with your configurator.
One more question:
Do you think the code could be adapted easily to an Akai mpc mini controller?
The APC has a 64 non-modifiable buttons sending notes on ch1 (it's definitely a pity...) 

Have a good afternoon.

[midimix gab 100ctrl_ch9 with toggle.sbr.zip](https://github.com/user-attachments/files/18673074/midimix.gab.100ctrl_ch9.with.toggle.sbr.zip)

**Reply by sonotor** [reply]
Sorry I've just discovered that bank left solo button nr.1 is not transmitting the right data....

**Reply by sonotor** [reply]
Hi Stefan,
I hope this message will find you well.
One more little bug.... One knob stays for ever on midi ch 1...
Detail on the screenshot in attachment.
merci beaucoup
![Capture d’écran 2025-02-19 à 17 19 04](https://github.com/user-attachments/assets/c0a2125e-a904-4384-a940-1603db05e8dd)
